tests/README.md

```diff
@@ -1,28 +1,62 @@
-# Connect integration tests
+# trezor-connect tests
+## Continuous Integration
+Connect tests are running in two environments:
+- [SatoshiLabs GitLab CI](https://gitlab.com/satoshilabs/trezor/connect/-/pipelines) NixOS runner (trezor-user-env)
+- [Github Actions CI](https://github.com/trezor/connect/actions) Ubuntu runner
```

Comment:
Maybe we could elaborate that in Gitlab we run directly inside trezor-user-env while GH Actions uses the image via `/.run.sh` therefore kind of "mimicks" when run locally?

Reply:
7255f9f

Reply:
i guess i can resolve this